src/main/java/com/android/volley/toolbox/BasicNetwork.java

```diff
@@ -65,6 +66,9 @@
 
     protected final ByteArrayPool mPool;
 
+    private long mRequestStart;
+    private ProgressListener mProgressListener;
```

Comment:
@alexpopa95 qui ti chiederei di effettuare alcune modifiche.

Non è necessario che il progress listener sia definito come variabile. E' meglio che sia utilizzato localmente.

Fai l'overload del metodo `entityToBytes`, o, se la cosa non comporta problemi, modifica la signature in modo che prenda in ingresso il progress listener.

Dato che il progress listener è implementato a livello di request, il progress può essere notificato alla sola request e non mischiamo troppo le cose.